src/tryParseJson.js

```diff
@@ -11,3 +9,27 @@ import {createSafeJsonParser} from '@sanity/util/createSafeJsonParser'
 export const tryParseJson = createSafeJsonParser({
   errorLabel: 'Error streaming dataset',
 })
+
+function createSafeJsonParser({errorLabel}) {
```

Comment:
thanks for inlining this – didn't seem worth the extra dependency :)  (guess it made more sense when both packages were in the same monorepo)